common/kernel/command.cc

```diff
@@ -189,6 +189,8 @@ po::options_description CommandHandler::getGeneralOptions()
     general.add_options()("placer-heap-critexp", po::value<int>(),
                           "placer heap criticality exponent (int, default: 2)");
     general.add_options()("placer-heap-timingweight", po::value<int>(), "placer heap timing weight (int, default: 10)");
+    general.add_options()("placer-heap-cell-placement-timeout", po::value<int>(),
+            "allow placer to attempt up to the given number of iterations to place the cell (int, default: design size^2 / 8, 0 for no timeout)");
```

Comment:
I believe your idea to control in terms of array (squared) is nice, but why not allow user control the divisor ?

```
  general.add_options()("placer-heap-cell-placement-timeout", po::value<int>(),
     "allow placer to attempt up to (design_size^2 / N) of iterations to place the cell 
      (int N, default: 8, 0 for no timeout)");
```

* So in this way we could end up: 
      N = 8 default, otherwise what user input,
   *  if N > 0 : (design_size^2) / N 
   *  if N = 0 : no-timeout

Reply:
Oh hmm.. sure, I hadn't considered that.

Reply:
Third time is a charm! This is much better than an absolute value, allowing a user to clamp down or run unlimited. Apologies for being slow to catch on here.